2007-03-18  Joseph Myers  <joseph@codesourcery.com>

	* configure.in: Also pass -isystem option for GCC's include-fixed
	directory.
	* configure: Regenerate.

Index: configure.in
===================================================================
RCS file: /cvs/glibc/libc/configure.in,v
retrieving revision 1.468
diff -u -r1.468 configure.in
--- configure.in	17 Mar 2007 17:01:46 -0000	1.468
+++ configure.in	18 Mar 2007 20:35:03 -0000
@@ -912,7 +912,7 @@
 # thing on a system that doesn't need fixincludes.  (Not presently a problem.)
 if test -n "$sysheaders"; then
   ccheaders=`$CC -print-file-name=include`
-  SYSINCLUDES="-nostdinc -isystem $ccheaders \
+  SYSINCLUDES="-nostdinc -isystem $ccheaders -isystem $ccheaders-fixed \
 -isystem `echo $sysheaders | sed 's/:/ -isystem /g'`"
   if test -n "$CXX"; then
     cxxversion=`$CXX -dumpversion 2>&AS_MESSAGE_LOG_FD` &&

--
Joseph S. Myers
joseph@codesourcery.com
